Assets/Scripts/Common Sripts/DestroyGround.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Outscal.BattleTank
+{
+    public class DestroyGround : MonoBehaviour
+    {
+        public GameObject[] military;
+        public GameObject[] oilFields;
+        public GameObject[] rocks;
+
+        public GameObject completeLevelArt;
+        public GameObject lit;
+
+
+        void Start()
+        {
+            DestroyMilitary();
+        }
+
+        async void DestroyMilitary()
```

Comment:
Give function definition before every function so that we get to know what exactly function is doing